model-api/src/commonTest/kotlin/org/modelix/model/api/BuiltinLanguagesTest.kt

```diff
@@ -41,4 +43,15 @@ class BuiltinLanguagesTest {
         // They were only accessible by directly calling Model.modelImports for example.
         assertEquals(3, childLinks.size)
     }
+
+    @Test
+    fun allBuildInLanguagesHaveMpsConceptId() {
+        val concepts = BuiltinLanguages.getAllLanguages()
+            .flatMap { it.getConcepts() }
+
+        concepts.forAll { concept: IConcept ->
+            concept.getReference()
```

Comment:
Is there an assertion missing related to the reference or does kotest implicitly do some magic here?